docs/reference/decision_points/human_impact.md

```diff
@@ -51,8 +51,6 @@ from ssvc.decision_tables.helpers import dt2df_md
 print(dt2df_md(DT))
 ```
 
-
-[^1]: In pilot implementations of SSVC, we received feedback that organizations tend to think of mission and safety impacts as
 if they were combined into a single factor: in other words, the priority increases regardless which of the two  impact factors was increased.
```

Comment:
This paragraph shows a deletion that leaves it without a beginning to the sentence.